code/network/multimsgs.h

```diff
@@ -497,6 +498,10 @@ void process_NEW_primary_fired_packet(ubyte* data, header* hinfo);
 void send_non_homing_fired_packet(ship *shipp, int banks_fired, bool secondary = false);
 void process_non_homing_fired_packet(ubyte *data, header *hinfo);
 
+// animation triggered info
+void send_animation_triggered_packet(int animationId, int pmi, const animation::ModelAnimationDirection& direction, bool force, bool instant, const int* time = nullptr);
+void process_animation_triggered_packet(ubyte* data, header* hinfo);
```

Comment:
I'm not fully up on all of the code so this might be a pointless question, but just to clarify, this packet is only for starting an animation. What about when an animation needs to stop or change direction? Is that handled in the older packets, and if so then why is this new one needed? If not then this packet should be modified now to handle those other cases if needed rather than changing it later or adding additional packets for that purpose.